pennylane/ftqc/pauli_tracker.py

```diff
@@ -117,24 +128,74 @@ def pauli_prod(ops: List[Operator]) -> Operator:
             from pennylane.ftqc.pauli_tracker import pauli_prod
             from pennylane import I, X, Y, Z
             >>> pauli_prod([I(0),X(0),Y(0),Z(0)])
-            I(0)
+            (0, 0)
 
         A Pauli operator is returned for a list of Pauli operator up to a global phase.
     """
 
     if len(ops) == 0:
         raise ValueError("Please ensure that a valid list of operators are passed to the method.")
-    op0 = ops.pop()
-    res_x, res_z = pauli_to_xz(op0)
-    op0_wire = op0.wires
+    res_x, res_z = pauli_to_xz(ops.pop())
 
     while len(ops) > 0:
-        op = ops.pop()
-        wire = op.wires
-        if wire != op0_wire:
-            raise ValueError("All operators should target at the same wire.")
-        x, z = pauli_to_xz(op)
+        x, z = pauli_to_xz(ops.pop())
         res_x ^= x
         res_z ^= z
 
-    return xz_to_pauli(res_x, res_z)(wires=op0_wire)
+    return (res_x, res_z)
+
+
+def apply_clifford_op(
+    clifford_op: Operator, xz: List[Tuple[np.uint8, np.uint8]]
+) -> List[Tuple[np.uint8, np.uint8]]:
+    """Commuting a list of Pauli ops to a list of new Pauli ops with a given Clifford op.
```

Comment:
Not exactly sure of the intended meaning of this docstring --- can you elaborate a little more on it?
Is it commutation of an operation past/through other operations?

Reply:
More explanations (including a math equation) added.